docs/getting-started.md

```diff
@@ -0,0 +1,50 @@
+---
+id: getting-started
+title: "Getting Started"
+---
+
+## Add the dependency to your build.sbt file
+
+```scala
+libraryDependencies += "dev.zio" %% "zio-dynamodb" % @VERSION@
+```
+
+### Read & write data to/from DynamoDB
+
+```scala
```

Comment:
Not necessary, but it's good to have compile-time safety on code blocks in the documentation, e.g.

```scala mdoc:compile-only
code snippet
```

Reply:
cool, lets do this in a separate PR then